weekend_scraper/scraper.py

```
@@ -0,0 +1,89 @@
+    Filters for outbound Fri >= 17:00 and inbound Sun >= 17:00.
+    """
+    def get_oneway(date_obj, src, dest):
+        q = create_query(
+            flights=[FlightQuery(date=date_obj.strftime("%Y-%m-%d"), from_airport=src, to_airport=dest)],
+            trip="one-way", language="en-GB", currency="GBP"
+        )
+        # Retry logic
+        for attempt in range(3):
+            try:
+                res = get_flights(q, integration=integration)
+                if res: return res
```

Ruff (0.16.0)

[error] Multiple statements on one line (colon)

(E701)